.github/workflows/automate_issue_labels.yml

```diff
@@ -9,9 +9,39 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      - name: Check for Team label inclusion
+        run: |
+          if ${{ contains(github.event.issue.labels.*.name, 'Team/DevTools') || contains(github.event.issue.labels.*.name, 'Team/Observability') ||
+              contains(github.event.issue.labels.*.name, 'Team/jBallerina') ||
+              contains(github.event.issue.labels.*.name, 'Team/LanguageServer') ||
+              contains(github.event.issue.labels.*.name, 'Team/CompilerFETools') ||
+              contains(github.event.issue.labels.*.name, 'Team/CompilerFE') ||
+              contains(github.event.issue.labels.*.name, 'Team/StandardLibs') ||
+              contains(github.event.issue.labels.*.name, 'Team/VSCode') ||
+              contains(github.event.issue.labels.*.name, 'Team/CICD') ||
+              contains(github.event.issue.labels.*.name, 'Team/Website') ||
+              contains(github.event.issue.labels.*.name, 'Team/LanguageServer/Extensions') }} == true; then
```

Comment:
Can we replace this with a single check for the `Team/`  substring instead? It'll also be more future proof incase more new team label types will be added to the repo.

Reply:
Tried with both `Team/` and `Team/*` substrings but was not working with any of these. glob patterns seems not supporting with `contains` method